### PR TITLE
Corrected colophon

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: 1001-2000/LIT1287Dersan.xml [hunk 1]
```diff
@@ -90,12 +90,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
        
          </div>
          <div type="edition" xml:lang="gez">
-            <div  type="textpart" subtype="incipit"><ab>ድርሳን፡ ዘደረሰ፡ አብ፡ ብፁዕ፡ ኤጴስ፡ ቆጶስ፡ (sic) ክቡር፡ አባ፡ ቴዎዶጦስ፡ ኤጴስቆጶስ፡ ዘዕንቆራ፡ 
+            <div type="textpart" subtype="chapter" xml:id="Introduction"><label>Introduction</label>
+               
+               <div  type="textpart" subtype="incipit"><ab>ድርሳን፡ ዘደረሰ፡ አብ፡ ብፁዕ፡ ኤጴስ፡ ቆጶስ፡ (sic) ክቡር፡ አባ፡ ቴዎዶጦስ፡ ኤጴስቆጶስ፡ ዘዕንቆራ፡ 
                እምአድያመ፡ ገላትያ፡ በዕለተ፡ ተዝካሩ፡ ለፀሐየ፡ ጽድቅ፡ ክቡር፡ ወልዑል፡ ኮከበ፡ ጽባሕ፡ ብሩህ፡
                ወጽዱል፡ ወዝንቱሰ፡ አቡነ፡ ብፁዕ፡ ቴዎዶጦስ፡ ዘነበበ፡ ድርሳን፡ ገድል፡ እንተ፡ ቅዱስ፡ ማር፡ ጊዮርጊስ፡ ወውእቱኬ፡
                ፩፡ እምአበዊነ፡  ኤጴስቆጶሳት፡ ቅዱሳን፡ እለ፡ ተጋብዑ፡ በኒቅያ፡ በእንተ፡ ሃይማኖት፡ ርትዕት፡
```
Comment (DenisNosnitsin1970) [inline 1001-2000/LIT1287Dersan.xml]: ኤጲስቆጶሳት?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: 1001-2000/LIT1287Dersan.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1287Dersan.xml]: ኤጲስቆጶስ?

**Comment by eu-genia** [comment]
this is so old I guess the branch is long obsolete, there have been so many changes on top... will take a while now

File: 1001-2000/LIT1287Dersan.xml (hunk 1 above)
Comment (eu-genia) [reply 1001-2000/LIT1287Dersan.xml]: I cannot understand where these corrections belong, sorry. this is a very very old PR

Comment (eu-genia) [reply 1001-2000/LIT1287Dersan.xml]: Ok got it

**Comment by eu-genia** [comment]
Denis can this be merged?

**Pull request merged**